### PR TITLE
var -> let

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/ecmascript-2015/function-improvements/property-value-shorthand.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/ecmascript-2015/function-improvements/property-value-shorthand.md?ref=3fbad695d7b3bac1e5b12ef741ea278411dba46e)

File: javascript/ecmascript-2015/function-improvements/property-value-shorthand.md [hunk 1]
```diff
@@ -13,8 +10,6 @@ aspects:
   - new
   - workout
 
-inAlgoPool: false
-
 links:
   - '[benmvp.com](http://www.benmvp.com/learning-es6-enhanced-object-literals/){website}'
```
Comment (kapnobatai137) [inline javascript/ecmascript-2015/function-improvements/property-value-shorthand.md]:
```suggestion
  - '[Enhanced Object Literals](http://www.benmvp.com/learning-es6-enhanced-object-literals/){website}'
```

File: javascript/ecmascript-2015/function-improvements/property-value-shorthand.md [hunk 2]
```diff
@@ -13,8 +10,6 @@ aspects:
...
   - '[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer){website}'
```
Comment (kapnobatai137) [inline javascript/ecmascript-2015/function-improvements/property-value-shorthand.md]:
```suggestion
  - '[Object Initializer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer){documentation}'
```

**Comment by kapnobatai137** [comment]
image

added a code block example of the thrown exception. will also check the previous insight to see if we contradict ourselves

**Pull request merged**